nullaway/src/test/java/com/uber/nullaway/NullAwayCoreTests.java

```diff
@@ -709,4 +710,40 @@ public void staticCallZeroArgsNullCheck() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void allowLibraryModelsOverrideAnnotations() {
```

Comment:
Maybe worth it to have test cases for this in a new test class, rather than `NullAwayCoreTests`? Maybe a `NullAwayCustomLibraryModelsTests.java` (extending `NullAwayTestsBase`) which could:

1. Have a general method to get a test helper with `TestLibraryModels.class` already in the processor path: `makeLibraryModelsTestHelperWithArgs`
2. Include this test case and its converse (see comment below) in that new class.
3. Move `libraryModelsOverrideRestrictiveAnnotations` from `NullAwayAcknowledgeRestrictiveAnnotationsTests` to this new class, so we have all the test cases that load `TestLibraryModels.class` into `-processorpath` in the same place.

Reply:
54a149c1568e92cef295e67397486b3af08db350